source/_integrations/eheimdigital.markdown

```diff
@@ -151,6 +149,10 @@ actions:
 
 {% enddetails %}
 
+## Known limitations
+
+- The integration does not support other EHEIM Digital devices than those listed above. More devices will be added in future updates. It is, however, supported to have an unsupported device as the main device and supported devices as sub-devices, even though the unsupported device will not have any entities shown in Home Assistant.
```

Comment:
I find this a bit confusing. Could make an example, maybe?

Reply:
Ah, sorry. With EHEIM Digital, all devices can take the role of main/master or sub/slave. It just depends on which was set up at first, which becomes the main device, and the others are the subs.

If you have for example three devices, from which one is unsupported but set up as the main device, the integration works and communicates with all devices via the main device (the devices have a "meshing" so you talk to the main device and the messages get forwarded and vice versa) but the unsupported main device has no entities.

Reply:
Ok, thank you. That makes it a bit clearer. But I still don't understand.. why would I want add a device if I cannot see its entities?

Reply:
You add the system via its IP address (or not even that, because in normal network setups it is addressable at `eheimdigital.local`) and all devices are added automatically.